premerge/premerge-advisor.md

```diff
@@ -0,0 +1,168 @@
+# Premerge Advisor
+
+## Introduction
+
+While the premerge infrastructure is generally reliable, tests that are broken
+at HEAD, or tests that are flaky can significantly degrade the user experience.
+Failures unrelated to the PR being worked on can lead to warning fatigue which
+can compound this problem when people land failing tests into main that they
+believe are unrelated. The premerge advisor is designed to run after premerge
+testing and signal to the user whether the failures can be safely ignored because
+they are flaky or broken at head, or whether they should be investigated further.
+
+## Background/Motivation
+
+The usefulness of the premerge system is significantly impacted by how much
+people trust it. People trust the system less when the issues reported to them
+are unrelated to the changes that they are currently working on. False positives
+occur regularly whether due to flaky tests, or due to main being broken. Efforts
+to fix these issues at the source and keep main always green and deflake tests
+are laudable, but not a scalable solution to the problem of false positive
+failures. It is also not reasonable to expect PR authors to spend time
```

Comment:
suggested changes:  "laudable, but not a scalable solution to"  => "ongoing, but are not sufficient to prevent" and  "failures." => "failure reports."

Reply:
Changed from laudable to ongoing.

I think it's still important to mention that this is a scalability problem. And fixing flaky tests fixes the failures (which is ideal) rather than just making them go unreported.